docs/lbl/lbl.rst

```diff
@@ -186,6 +186,56 @@ with its built-in :ref:`spectroscopic constants <label_db_spectroscopic_constant
 or supply an energy level database. In the latter case, you need to edit the 
 :ref:`Configuration file <label_lbl_config_file>` . 
 
+Calculating spectrum using GPU
+------------------------------
+
+RADIS supports calculation of spectra at thermal equilibrium using GPU for the
+calculation of lineshapes and broadening. If your system supports it, the spectrum
+can be calculated on the GPU using the :py:func:`~radis.lbl.calc.calc_spectrum`
+function with parameter `mode` set to `gpu`.
+
+Currently, the GPU-enabled spectrum calculations can only be done using databank
+that has been preprocessed and saved in numpy's `npy` format. Direct .par or .txt
+files *cannot* be used for calculating spectra on the GPU.
+
+The `npy` files that are needed for calculating the spectra on GPU can be extracted
+from any databank and stored in the following format. The name of the file is written
+first, followed by the physical quantity it stores and it's name and position in the
+CDSD-4000 database.
+
+`v0.npy`: wavenumber in vacuum; `v0`, line[3:15]
+`da.npy`: air-pressure induced shift; `d_air`, line[59:67]
+`El.npy`: low-state energy; `Elow`, line[45:55]
+`na.npy`: temperature dependence exponent for air; `n_air`, line[55:59]
+
+In addition to the above 4 quantities, we also need 3 more quantities which are not
+directly stored in the databank. They are explained below:
+
+`log_2gs.npy`: np.log(2*gs), where `gs` is HITRAN/HITEMP HWHM pressure broadening constant
+for self-broadening; `gamma_self`, line[40:45]
+`log_2vMm.npy`: np.log(2*v0) + 0.5*np.log(2*k*np.log(2)/(c**2*Mm)), where `v0` is the
+wavenumber in vacuum, `k` is Boltzmann's constant, `c` is speed of light in vacuum
+and `Mm` is the molecular mass of gas molecule in kilogram.
+`S0.npy`: f_ab * gu * A21 / (8*pi*c_cm*v0**2) where,
+`f_ab`: np.array([ 0.98420, 0.01106, 0.0039471])[iso.astype(int)-1],
+`gu`: 2*Ju + 1, where
+`Ju` = Jl + DJ, where
+`DJ` = ord(line[117:118])-ord('Q')
+`Jl` = int(line[118:121])
+`A21` is the Einstein's coefficient, line[25:35]
+`c_cm` is speed of light in vacuum in centimeters/second,
+`v0` is wavenumber in vacuum.
+
+In order to facililate the conversion of data from the CDSD-4000 par format to the format explained
+above, users can use the scripts present in `/radis/misc/prepare-npy-data`.
+
+Note that the data
```

Comment:
Typo here:

```suggestion
Note that the data
```

Reply:
Also you could skip this bit for the time being. This example was an (incomplete) part of the other branch that I merged with the gpu branch. I was working on that since the original one had failed because of the `npy2df` commit. I'll finish the example soon so you can come back to this once that's done.